docs/examples/snpsift.md

```diff
@@ -0,0 +1,22 @@
+# Annotate SNPSift Files
+
+The script [annotate_input_files.py](./annotate_snpsift.py) can be used to match variants from input files to an instance of GraphKB. Copy the script locally and install the package dependencies.
```

Comment:
The script is `annotate_snpsift.py` but referred to as `annotate_input_files.py` here. Is there a reason for that?

Reply:
ah, name change and I forgot to update it. good catch